test-worker/manifest.yml

```diff
@@ -5,3 +5,6 @@ packages:
       worker:
         function: worker.js
         docker: 'alexkli/nodejs10-action-centos:2.4.1'
+      limits:
+        memorySize: 4096
```

Comment:
bmp with transparency -> gif (which supports transparency) seems to be more memory intensive than I'd had expected